src/components/HomePage.jsx

```diff
@@ -180,6 +180,14 @@ class HomePage extends Component {
 					"Conda package": "https://anaconda.org/in-core/pyincore-viz",
 				}
 			},
+			{
+				title: "pyIncore-data",
+				repoName: "pyincore-data",
+				options: {
+					"Change log": "https://github.com/IN-CORE/pyincore-data/blob/master/CHANGELOG.md",
+					"GitHub": "https://github.com/IN-CORE/pyincore-data"
+				}
```

Comment:
Is there conda package for pyincore data? If so could you add that? you can see line 180 for example

Reply:
yes, there is. Added it. Good catch, thanks.